ftw/publisher/sender/browser/configlet.py

```diff
@@ -502,6 +494,27 @@ def get_translated_action(self):
         return TRANSLATED_ACTIONS.get(self.job.action, self.job.action)
 
 
+class PublisherConfigAction(PublisherConfigletView):
+
+    def __call__(self, *args, **kwargs):
+        view = None
+
+        if self.request.get('button.cleanJobs'):
+            view = '@@publisher-config-cleanJobs'
+        if self.request.get('button.executeJobs'):
+            view = '@@publisher-config-executeJobs'
+        elif self.request.get('button.execute'):
+            view = '@@publisher.execute.job'
+        elif self.request.get('button.delete'):
+            view = '@@publisher.remove.job'
+        elif self.request.get('requeue.job'):
+            view = '@@publisher-config-listExecutedJobs'
+
+        if not view:
+            return self.request.RESPONSE.redirect('@@publisher-config')
+        return self.context.restrictedTraverse(view)()
```

Comment:
Why did you choose to have use a proxy view?

I'd have done it like this:
- Use a form for each link / button. The form-action points to the view name as the link did before.
- Protect the view so that it only accepts POST.
- Use the `plone.protect` token verification
  - it is important that we do that so that the token is verified _before_ the request is executed, since some views modify the file system and are not transaction save
  - this adds CSRF-protection even when auto-CSRF is disabled

Example: https://github.com/4teamwork/ftw.trash/blob/3e9dcfd33a78ac908218c37fdd18f3d253946b47/ftw/trash/browser/trash.py#L36-L37
See also https://github.com/plone/plone.protect